Troupon.Catalog.Api/Controllers/LocationController.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using Infra.Api;
+using Infra.Api.Conventions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace Troupon.Catalog.Api.Controllers
+{
+  [ApiController]
+  [Route("api/v{version:apiVersion}/[controller]")]
+  [ApiVersion("1.0")]
+  [ApiConventionType(typeof(PwcApiConventions))]
+  public class LocationController : PwcBaseController
```

Comment:
Do you want to have Location**S** as other controllers ?

Reply:
Yes! Good catch!

Reply:
🐠